ui/js/controllers/logviewer.js

```diff
@@ -210,11 +210,29 @@ logViewerApp.controller('LogviewerCtrl', [
                 });
             });
 
+            // Make the log and job artifacts available
             ThJobArtifactModel.get_list({job_id: $scope.job_id, name__in: 'text_log_summary,Job Info'})
             .then(function(artifactList) {
                 artifactList.forEach(function(artifact) {
                     if (artifact.name === 'text_log_summary') {
                         $scope.artifact = artifact.blob;
+                        $scope.step_data = artifact.blob.step_data;
+                        var initialStep = $scope.artifact.step_data.steps[0];
```

Comment:
So here's the tweak for those strange TC log artifacts described in the PR conversation.